ruby/ext/google/protobuf_c/extconf.rb

```diff
@@ -38,3 +54,12 @@
          "shared_message.c"]
 
 create_makefile(ext_name)
+
+if debug_symbols_output_dir
+  File.open("Makefile", "a") do |o|
+    o.puts "generate_dbg_file: $(DLLIB)"
+    o.puts "\t$(ECHO) Saving debug symbols in #{debug_symbols_output_dir}/#{debug_symbols}"
+    o.puts "\t$(Q) objcopy --only-keep-debug $(DLLIB) #{debug_symbols_output_dir}/#{debug_symbols}"
+    o.puts "all: generate_dbg_file"
```

Comment:
I wanted to see what the previous `all` was, I wonder if we should add a comment like
```suggestion
    # The previous all was "$(DLLIB)" so this just inserts a step.
    o.puts "all: generate_dbg_file"
```